routes.js

```diff
@@ -12,3 +12,5 @@ routes
   .add('manage group', '/groups/manage', '/groups/manage')
   .add('group search', '/groups/search', '/groups/search')
   .add('group profile', '/groups/:id', '/groups/group')
+  .add('user profile', '/users/:id', '/users/user')
+  .add('user profile currentUser', '/:id(my)/profile', '/users/user')
```

Comment:
So I know what you're thinkin. Yes this works since my state mapper corrects `my` to the current user's ID, and the router only picks this up if :id is equal to `my`

Reply:
Dayum. that's actually really cool. :-D

Reply:
the side effect is that `/users/my` leads to the same result, but that won't be an issue if we get uuid pattern matching working.